content/800-data-platform/200-projects/110-create-new-application-from-template.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Create a new project from a template'
+metaTitle: 'Create a new Node.js application project from a template in the Prisma Data Platform'
+metaDescription: 'If you want to explore how to create projects in the Prisma Data Platform, you can start with template Prisma schema which is immediately migrated to an empty database.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+If you want to explore how to create projects in the Prisma Data Platform, you can start with template Prisma schema which is immediately migrated to an empty database.
+</TopBlock>
+
+## Prerequisites and considerations
+
+* GitHub account
+* [Sign up for the Prisma Data Platform](/data-platform/accounts#sign-up-for-the-prisma-data-platform)
+* Connection string to a [Prisma ORM-supported databases](/reference/database-reference/supported-databases)
+* [Grant the Prisma GitHub app read and write permissions to your GitHub repositories](/data-platform/projects#grant-read-and-write-permissions-to-the-prisma-github-app)
+
+## Steps
+
+1. Go to the [_Projects_](https://cloud.prisma.io/projects) page and click **New Project**.
+2. On the _Configure project_ page, name your project and create a new GitHub repository.
+    1. In the **Display Name** text box, provide a name for your project.
+        <Admonition>
+        
+        **Note**<br /><br />
+        The display name of a project appears in several places in the Prisma Data Platform.<br />
+          &nbsp;• [_Projects_](https://cloud.prisma.io/projects) page<br />
+          &nbsp;• [_Billing_](https://cloud.prisma.io/billing) page<br />
+          &nbsp;• **Projects** drop-down in the upper-left
+        </Admonition>
+    2. From the **GitHub Accounts & Organizations** drop-down menu, select a GitHub account or an organization.
```

Comment:
You might disagree here, but in general IMO it's better to avoid the names of control types in GUIs. So you might say:

"From **GitHub Accounts & Organizations**, select..."

In other words, omit "drop-down menu". For example, the Microsoft Style Guide says "Don’t include the word menu unless it adds needed clarity." (It has similar guidelines for other GUI elements). I turned to the MS Style Guide because I couldn't find a guideline on GUI elements in the Rackspace Style Guide (Tana's preferred baseline style guide).

I'd also make a similar comment about the other UI elements in this procedure.

However, this is a style policy we will need to agree on as a team. This review probably isn't the place to discuss it! So I'm not suggesting you should change it right now - I'm just mentioning this as a talking point.